src/btree/bt_page.c

```diff
@@ -151,9 +154,13 @@ __page_inmem_prepare_update(WT_SESSION_IMPL *session, WT_ITEM *value, WT_CELL_UN
         WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, &size));
         total_size += size;
         tombstone->durable_ts = WT_TS_NONE;
-        tombstone->start_ts = unpack->tw.stop_ts;
         tombstone->txnid = unpack->tw.stop_txn;
         tombstone->prepare_state = WT_PREPARE_INPROGRESS;
+        tombstone->start_ts = unpack->tw.stop_ts;
```

Comment:
same here.

Reply:
as discussed, since we added two prapre_ts and prepared_id fields in wt_time_window, when writing to disk, we should keep the reading from disk here agnostic of the flags